src/components/Rules.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import './Rules.css'
+import App from './App.js';
+
+class Rules extends React.Component {
+    constructor(props) {
+        super(props)
+      }
+
+    render() {
+
+        return (<div>
+        <h1 className = "Heading">Scrabble Rules</h1>
+        <div className = "rules">
+        <p>The basic objective of Scrabble is to play tiles marked with letters on a 15x15 grid to form words. After the initial word is played, players take turns adding words to existing letters.</p>
+        <p> Games only have two players. Each match has two games, one with each player playing first. </p>
```

Comment:
We might not be following this rule, but it's fine, no need to change right now, I'm just pointing it out.

Reply:
But It looks great.

Reply:
No, no, I just meant the rule which said that a game only has two players and each match having two games. ( We were thinking of more than two people playing in the same room as well, informal fun games)

Reply:
Ohh 😅😅😅